tests/Resources/Common.robot

```diff
@@ -80,3 +81,16 @@ Get Cluster Name By Cluster ID
         Fail    Unable to retrieve cluster name for cluster ID ${cluster_id}
     END
     [Return]    ${cluster_name}
+
+Get Notification Email From Addon-Managed-Odh-Parameters Secret
+    [Documentation]    Gets email form addon-managed-odh-parameters secret
+    ${resp} =    Oc Get  kind=Secret  namespace=redhat-ods-operator  name=addon-managed-odh-parameters
+    ${resp} =  Evaluate  dict(${resp[0]["metadata"]["annotations"]["kubectl.kubernetes.io/last-applied-configuration"]})
+    [Return]  ${resp["stringData"]["notification-email"]}
+
+Check Email Is Present In Alertmanager ConfigMap
+    [Documentation]    Check expected email is present in Alertmanaher
+    [Arguments]        ${email_to_check}
+    ${resp} =    Oc Get  kind=ConfigMap  namespace=redhat-ods-monitoring  name=alertmanager
+    ${resp} =  Evaluate  str(${resp[0]["metadata"]["annotations"]["kubectl.kubernetes.io/last-applied-configuration"]})
+    Should Contain    ${resp}    ${email_to_check}
```

Comment:
It should look into the user-notifications field, not the whole alertmanager file

```
    - name: 'user-notifications'
      email_configs:
      - to: 'dummyEmail@redhat.com'
        send_resolved: false
        html: '{{ template "email.rhods.html" . }}'
        headers: 
          subject: '{{ template "email.rhods.subject" . }}'
```

Reply:
Done